qa-tests-backend/src/main/groovy/util/Env.groovy

```diff
@@ -8,30 +9,31 @@ import java.nio.file.attribute.FileTime
 import org.slf4j.Logger
 import org.slf4j.LoggerFactory
 
+@CompileStatic
 class Env {
 
     private static final Logger LOG = LoggerFactory.getLogger(this.getClass())
 
-    private static final PROPERTIES_FILE = "qa-test-settings.properties"
+    private static final String PROPERTIES_FILE = "qa-test-settings.properties"
 
-    private static final DEFAULT_VALUES = [
+    private static final Map<String, String> DEFAULT_VALUES = [
             "API_HOSTNAME": "localhost",
             "API_PORT": "8000",
             "ROX_USERNAME": "admin",
     ]
 
-    static final IN_CI = (System.getenv("CI") == "true")
-    static final CI_JOB_NAME = System.getenv("CI_JOB_NAME")
-    static final BUILD_TAG = System.getenv("BUILD_TAG")
-    static final GATHER_QA_TEST_DEBUG_LOGS = (System.getenv("GATHER_QA_TEST_DEBUG_LOGS") == "true")
-    static final QA_TEST_DEBUG_LOGS = System.getenv("QA_TEST_DEBUG_LOGS") ?: ""
-    static final HAS_WORKLOAD_IDENTITIES = (System.getenv("SETUP_WORKLOAD_IDENTITIES") == "true")
+    static final boolean IN_CI = (System.getenv("CI") == "true")
+    static final String CI_JOB_NAME = System.getenv("CI_JOB_NAME")
+    static final String BUILD_TAG = System.getenv("BUILD_TAG")
+    static final boolean GATHER_QA_TEST_DEBUG_LOGS = (System.getenv("GATHER_QA_TEST_DEBUG_LOGS") == "true")
+    static final String QA_TEST_DEBUG_LOGS = System.getenv("QA_TEST_DEBUG_LOGS") ?: ""
+    static final boolean HAS_WORKLOAD_IDENTITIES = (System.getenv("SETUP_WORKLOAD_IDENTITIES") == "true")
 
     static final String IMAGE_PULL_POLICY_FOR_QUAY_IO = System.getenv("IMAGE_PULL_POLICY_FOR_QUAY_IO")
 
     // REMOTE_CLUSTER_ARCH specifies architecture of a remote cluster on which tests are to be executed
     // the remote cluster arch can be ppc64le or s390x, default is x86_64
-    static final REMOTE_CLUSTER_ARCH = System.getenv("REMOTE_CLUSTER_ARCH") ?: "x86_64"
+    static final boolean REMOTE_CLUSTER_ARCH = System.getenv("REMOTE_CLUSTER_ARCH") ?: "x86_64"
```

Comment:
This looks like the type here should be `String`, not `boolean`

The usage patterns I found are:
```
Env.REMOTE_CLUSTER_ARCH == "x86_64"
Env.REMOTE_CLUSTER_ARCH == "ppc64le"
Env.REMOTE_CLUSTER_ARCH == "s390x"
Env.REMOTE_CLUSTER_ARCH != "ppc64le"
Env.REMOTE_CLUSTER_ARCH != "s390x"
```
as well as a switch where the case labels are strings

Reply:
Oh, maybe that's why we have test failures I could not debug. So even with `CompileStatic` we can mess with types :disappointed: